File: take_u_forward/recursion/5_Multiple Recursion Calls.java
// Q1.  Fibonacci series

import java.util.Scanner;

public class Main {

    public static int fibo(int n) {
        if (n <= 1) {
            return n;
        }

        int last = fibo(n - 1);
        int slast = fibo(n - 2);

        return last + slast;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the value of n:\t");
        int n = sc.nextInt();

        System.out.println(fibo(n));

        sc.close();
    }
}

File: take_u_forward/recursion/5_Multiple Recursion Calls.cpp
// Q1.  Fibonacci series
#include <iostream>
using namespace std;

int fibo(int n){
    if(n<=1) return n;
    int last=fibo(n-1);
    int slast=fibo(n-2);
    return last+slast;
}
int main() {
        
    int n;
    cout<<"Enter the value of n:\t";
    cin>>n;
    
    cout<<fibo(n);
    
    return 0;
}